dz_2/src/Homework_OOP/Circle.java

```diff
@@ -0,0 +1,26 @@
+package Homework_OOP;
+
+public class Circle {
+    private static final double PI = 3.14;
+
+    private double radius;
+
+    public Circle(double radius){
+        this.radius = radius;
+    }
+
+    public void printToConsole(){
+        System.out.println("Our circle with parameters: " + "Radius = " + radius + " cm " + "Const PI = " + PI);
+
+    }
```

Comment:
дані константи я б не виводив вона буде для всіх однакова
```suggestion
    public void printToConsole(){
        System.out.println("Our circle with parameters: " + "Radius = " + radius + " cm ");

    }
```